src/animl/plot_boxes.py

```diff
@@ -0,0 +1,61 @@
+"""
+Module: animl.plot_boxes
+Functionality to draw bounding boxes and labels provided image DataFrame.
+"""
+# pylint: disable=R0914
+# pylint: disable=no-member
+
+import cv2
+
+
+def draw_bounding_boxes(row, box_number, image_output_path):
```

Comment:
So for instance (re: in-line pylint disabling) here you could do:

```
def draw_bounding_boxes(row, box_number,  # pylint: disable=too-many-locals
                                                 image_output_path):
```

(with corrected indentation, of course, :-) )